k8s/clusters/eu-central-1a/out/terraform/kubernetes.tf

```diff
@@ -0,0 +1,483 @@
+    Name                                                      = "k8s.eu-central-1a.mdn.mozit.cloud"
+    "kubernetes.io/cluster/k8s.eu-central-1a.mdn.mozit.cloud" = "owned"
+    "kubernetes.io/kops/role"                                 = "public"
+  }
+}
+
+resource "aws_route_table_association" "eu-central-1a-k8s-eu-central-1a-mdn-mozit-cloud" {
+  subnet_id      = "${aws_subnet.eu-central-1a-k8s-eu-central-1a-mdn-mozit-cloud.id}"
+  route_table_id = "${aws_route_table.k8s-eu-central-1a-mdn-mozit-cloud.id}"
+}
+
+resource "aws_security_group" "masters-k8s-eu-central-1a-mdn-mozit-cloud" {
+  name        = "masters.k8s.eu-central-1a.mdn.mozit.cloud"
+  vpc_id      = "${aws_vpc.k8s-eu-central-1a-mdn-mozit-cloud.id}"
+  description = "Security group for masters"
+
+  tags = {
+    KubernetesCluster                                         = "k8s.eu-central-1a.mdn.mozit.cloud"
+    Name                                                      = "masters.k8s.eu-central-1a.mdn.mozit.cloud"
+    "kubernetes.io/cluster/k8s.eu-central-1a.mdn.mozit.cloud" = "owned"
+  }
+}
+
+resource "aws_security_group" "nodes-k8s-eu-central-1a-mdn-mozit-cloud" {
+  name        = "nodes.k8s.eu-central-1a.mdn.mozit.cloud"
+  vpc_id      = "${aws_vpc.k8s-eu-central-1a-mdn-mozit-cloud.id}"
+  description = "Security group for nodes"
+
+  tags = {
+    KubernetesCluster                                         = "k8s.eu-central-1a.mdn.mozit.cloud"
+    Name                                                      = "nodes.k8s.eu-central-1a.mdn.mozit.cloud"
+    "kubernetes.io/cluster/k8s.eu-central-1a.mdn.mozit.cloud" = "owned"
+  }
+}
+
+resource "aws_security_group_rule" "all-master-to-master" {
+  type                     = "ingress"
+  security_group_id        = "${aws_security_group.masters-k8s-eu-central-1a-mdn-mozit-cloud.id}"
+  source_security_group_id = "${aws_security_group.masters-k8s-eu-central-1a-mdn-mozit-cloud.id}"
+  from_port                = 0
+  to_port                  = 0
+  protocol                 = "-1"
+}
+
+resource "aws_security_group_rule" "all-master-to-node" {
+  type                     = "ingress"
+  security_group_id        = "${aws_security_group.nodes-k8s-eu-central-1a-mdn-mozit-cloud.id}"
+  source_security_group_id = "${aws_security_group.masters-k8s-eu-central-1a-mdn-mozit-cloud.id}"
+  from_port                = 0
+  to_port                  = 0
+  protocol                 = "-1"
+}
+
+resource "aws_security_group_rule" "all-node-to-node" {
+  type                     = "ingress"
+  security_group_id        = "${aws_security_group.nodes-k8s-eu-central-1a-mdn-mozit-cloud.id}"
+  source_security_group_id = "${aws_security_group.nodes-k8s-eu-central-1a-mdn-mozit-cloud.id}"
+  from_port                = 0
+  to_port                  = 0
+  protocol                 = "-1"
+}
+
+resource "aws_security_group_rule" "https-external-to-master-0-0-0-0--0" {
```

Comment:
we block external traffic to master

Reply:
We can use a nacl here, current kube cluster doesn't have this atm but we can set that up